syft/format/internal/cyclonedxutil/versions.go

```diff
@@ -0,0 +1,66 @@
+package cyclonedxutil
```

Comment:
I considered moving the remaining helpers under internal, however, there is shared code with Grype, so a larger effort would be required to consolidate more helper code under internal. I considered adding under the existing exported helpers packages, however, I didn't want to make the API surface area larger than it needed to be.

This PR nudges the format API closer to keeping more things under internal that should not be shared, but doesn't go as far as taking all opportunities to use it. This will be incremental work and be completed before Syft 1.0.